packages/pds-ember/addon/components/pds/app/index.hbs

```diff
@@ -0,0 +1,29 @@
+{{! link should point to <main> (i.e., <App.Body>) }}
+<a
+  href="#content"
+  class="pdsSkipLink"
+  data-test-app-skip-link
+>
+  Skip to Content
+</a>
+
+<a id="top" data-test-app-top-anchor></a>
+
+<div
+  class="
+    pds-app
+    {{if @showDrawer 'has-open-drawer'}}
+  "
+  ...attributes
+  data-test-app
+>
+  {{yield (hash
+    Banner=(component 'pds/app/banner')
+    Body=(component 'pds/app/body')
+    Drawer=(component 'pds/app/drawer' open=@showDrawer)
```

Comment:
Automatically initializes the `@open` argument.

Reply:
Can we also yield `@shoDrawer` maybe as `drawerIsOpen` ? Just thinking there's likely cases where something should change in the layout of the app based on the state of the drawer, so it'd be nice to have access to that.

Reply:
Makes sense, but I'd rather not spend cycles on what-if's.  We can revisit when we have a use case that warrants its addition.